internal/source/code.go

```diff
@@ -15,6 +15,12 @@ type Edit struct {
 	OldFunc  func(string) int
 }
 
+type CommentSyntax struct {
```

Comment:
Handling comments seems like a source code thing, so I popped this in the `source` package. The new `CleanedComments` func I wrote depends on comment syntax, and we already have a func `StripComments` (which doesn't depend on comment syntax but I think that's just an oversight/bug).